### PR TITLE
Gallery post format

### PR DESCRIPTION
**Pull request by adiraomj**
Implemented enhancement #50 

Added new template for gallery format.
Retrieved first gallery shortcode using `get_post_gallery`.
Stripped the gallery from the content.
Built slider using flexslider.
Styled Gallery in a new scss file.

### CODE REVIEW

**Comment by Misplon** [comment]
Thanks @adiraomj. Code looks great.

Can you please, confirm this is still working for you in testing? It doesn't seem to be working for me at the moment.

File: inc/template-tags.php [hunk 1]
```diff
@@ -465,3 +465,25 @@ function siteorigin_unwind_display_icon( $type ) {
 	}
 }
 endif;
+
+if ( ! function_exists( 'siteorigin_unwind_strip_gallery' ) ):
```
Comment (Misplon) [inline inc/template-tags.php]: Quick space between the` ):`

Thanks :)

**Comment by adiraomj** [comment]
Done

**Comment by Misplon** [comment]
@adiraomj Ok working for me now. Do you perhaps want to move the bullet navigation up onto the images and make them white?

**Comment by adiraomj** [comment]
Yeah sure. I'll do that

**Comment by adiraomj** [comment]
Done. Please have a look.

**Pull request merged**